third_party/gfootball_engine/src/systems/graphics/rendering/opengl_renderer3d.cpp

```diff
@@ -791,6 +815,7 @@ void OpenGLRenderer3D::SetTextureMode(e_TextureMode textureMode) {
       mapping.glEnable(GL_TEXTURE_2D);
       break;
   }
+   */
```

Comment:
This method is never called and calls are pointless when we draw with shaders. 
https://stackoverflow.com/questions/14179497/is-it-necessary-to-call-glenablegl-texture-before-using-textures-in-opengl-2-1